assembly/elements/pages/PackageDetails.ts

```diff
@@ -0,0 +1,247 @@
+import { customElements } from '../../../node_modules/asdom/assembly/index'
+import { AspkgElement } from '../AspkgElement'
+
+class PackageDetails extends AspkgElement {
+	static observedAttributes: string[] = []
+
+	constructor() {
+		super()
+		this.useShadow = false
+	}
+
+	template(): string {
+		const hostSelector = this.shadowRoot != null ? ':host' : this.tagName.toLowerCase()
+
+		return /*html*/ `
+			<section
+				id="readme"
+				style="
+					background: #1a2634;
+					font-size: 31px;
+					height: auto;
+					width: auto;
+					margin-left: 40px;
+					margin-right: 40px;
+					margin-top: 107px;
+					margin-bottom: 40px;
+					border-radius: 10px;
+					border: 1px solid rgb(40, 45, 50);
+					padding-bottom: 128px;
+				"
+			>
+				<div
+					style="
+						position: absolute;
+						top: 107px;
+						bottom: 40px;
+						right: 40px;
+						bottom: 0px;
+						width: 28%;
+						height: 90%;
+						background: #1a2634;
+						border-radius: 0px;
+						font-weight: normal;
+						padding-top: 0px;
+						margin-top: 40px;
+						margin-bottom: 40px;
+						border-color: rgb(40, 45, 50);
+						border-right: 1px solid rgb(40, 45, 50);
+						border-left: 2px solid rgb(40, 45, 50);
+						padding-bottom: 0px;
+					"
+				>
+					<span style="padding-left: 15px; color: rgb(255, 255, 255); font-weight: bold; font-size: 18px">Install</span>
+					<p
+						style="font-size: 18px; color: rgb(255, 255, 255); margin-top: 15px; font-weight: bold; padding-left: 15px"
+					></p>
+					<p style="font-size: 18px; padding-left: 15px">
+						<span
+							id="pkg-install"
+							style="font-size: 18px; color: rgb(255, 255, 255); background: rgba(9, 11, 16, 0.48); border-radius: 3px"
+						>
+							-
+						</span>
+					</p>
+					<p style="color: rgb(255, 255, 255); font-size: 18px; font-weight: bold; padding-left: 15px; width: auto">
+						Downloads
+					</p>
+					<p
+						style="
+							color: rgb(255, 255, 255);
+							font-size: 18px;
+							font-weight: normal;
+							padding-left: 15px;
+							width: auto;
+							margin-bottom: 16px;
+						"
+						href="https://github.com/aspkg/example"
+					>
+						-
+					</p>
+					<p
```

Comment:
This markup is from BSS output, pretty messy with all the duplicated inline styles. I haven't cleaned this one up yet.

Reply:
Yeah, that's what happens for drag-drop builders. haha!